### ISSUE TITLE
How do I design  a distributed system by nng

### ISSUE THREAD

**Issue by lemonlittlebaby** [nanomsg/nng#1679]
I am a newer in web progroming. And I want to use nng to design a distributed system.How do I get start?And is there some more detailed docs or books to learn?

**Comment by phsilva** [comment]
Even though there are may differences, I still go back to the original ZeroMQ guide https://zguide.zeromq.org/ to learn about the patterns.

I am afraid nng doesn't have any books published yet, only the reference manual at https://leanpub.com/nngref2e.

**Comment by lemonlittlebaby** [comment]
> Even though there are may differences, I still go back to the original ZeroMQ guide https://zguide.zeromq.org/ to learn about the patterns.
> 
> I am afraid nng doesn't have any books published yet, only the reference manual at https://leanpub.com/nngref2e.

Thank you

**Comment by gdamore** [comment]
No code requests here (although we would like a better guide someday!)

**Comment by gdamore** [comment]
Yeah, I think I made the mistake of trying to structure the documentation like man pages.  I'll probably break that in the next major release, because honestly it's confusing for anyone not coming from a UNIX background.  I don't think many people are using the man pages these days anyway -- its easier to use the online docs.

**Comment by gdamore** [comment]
I've bumped a reference for that page on the README.

I do intend to significantly restructure the docs in the future to make them more approachable.  For now though, there is nothing else to do here, so I'm closing this ticket.
